src/components/custom-antd/Tooltip/Tooltip.tsx

```diff
@@ -9,7 +11,7 @@ type TooltipProps = {
 const Tooltip: React.FC<AntdTooltipProps & TooltipProps> = props => {
   const {customColor, ...rest} = props;
 
-  return <AntdTooltip color={customColor || Colors.greyBG} {...rest} />;
+  return <AntdTooltip color={customColor || Colors.greyBG} mouseEnterDelay={0.39} mouseLeaveDelay={0.1} {...rest} />;
```

Comment:
Are we sure this Tooltip isn't used anywhere else? I'm afraid we mess up delays in other places then the dates...

Maybe we should add a new prop which is something like `delay="slow"` which refers to these values for the date tooltips and default to the original setting (which is for sure controlled globally through antd) for everything else?

Reply:
The AntdTooltip in couple of places, but with the changes these values could be set as as defaults.

You can override them you can use mouseEnterDelay, mouseLeaveDelay for the specific Tooltip to behave differently.

If you can provide specific values for these delays for slow, normal, fast then I can follow your advice. 

For know I would leave as it is.